dockerize/dockerize.py

```diff
@@ -8,26 +13,28 @@
 import shutil
 import subprocess
 import tempfile
+from dockerize.depsolver import DepSolver
```

Comment:
Wouldn't it be better having
```python
from .depsolver import DepSolver
```
Then you won't have to make code changes if you where to change the namespace.
Another way could be
```python
from . import depsolver 

# And then later in the code
deps = depsolver.DepSolver()
```